app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java

```diff
@@ -1723,7 +1728,11 @@ private synchronized boolean areWeBonded(String hunt_address) {
     private void watchdog() {
         if (last_time_seen == 0) return;
         if (prefs.getBoolean("bluetooth_watchdog", false)) {
-            if ((JoH.msSince(last_time_seen)) > 1200000) {
+
+            int bt_wdg_timer = JoH.parseIntWithDefault(Pref.getString("bluetooth_watchdog_timer", "20"), 10, 20);
```

Comment:
What happens if the user sets this to something like `0` what will that do? I think this needs to be constrained with a minimum and maximum.

Reply:
ok, will do.